readme.md

```diff
@@ -54,7 +70,7 @@ console.log(results);
 
 Your Answer:
 ```text
-
+The result would need to be rendered using res.render instead of just a console.log.
```

Comment:
no.. `.find` is asynchronous and will be undefined without using a promise or callback